### PR TITLE
Add endOfMonth, mondayOfWeek, sundayOfWeek

### PR DESCRIPTION
**Pull request by lucacervello**

### CODE REVIEW

File: src/Onebip/DateTime/UTCDateTime.php [hunk 1]
```diff
@@ -423,6 +423,27 @@ public function startOfMonth()
         );
     }
 
+    public function endOfMonth()
+    {
+	return self::box(
+	    $this->toDateTime()->format('Y-m-t')
```
Comment (pasquinis) [inline src/Onebip/DateTime/UTCDateTime.php]: This `format` is hidden by the name of the function, let a new name of the function reveal it.

File: src/Onebip/DateTime/UTCDateTime.php [hunk 2]
```diff
@@ -423,6 +423,27 @@ public function startOfMonth()
...
+	);
+    }
+
+    public function startOfWeek()
```
Comment (pasquinis) [inline src/Onebip/DateTime/UTCDateTime.php]: not for all European people the week start with `monday`, be more explicit in the name of the function

File: src/Onebip/DateTime/UTCDateTime.php [hunk 3]
```diff
@@ -423,6 +423,27 @@ public function startOfMonth()
...
+    {
+	return self::box(
+	    $this->toDateTime()->format('monday this week')
+	);
+    }
+
+    public function endOfWeek()
```
Comment (pasquinis) [inline src/Onebip/DateTime/UTCDateTime.php]: not for all European people the week end with `sunday`, be more explicit in the name of the function

File: spec/unit/Onebip/DateTime/UTCDateTimeTest.php [hunk 4]
```diff
@@ -711,6 +711,50 @@ public function testStartOfMonthWillGiveTheFirstDay()
         ;
     }
 
+    public function testEndOfMonthWillGiveTheLastDay()
```
Comment (pasquinis) [inline spec/unit/Onebip/DateTime/UTCDateTimeTest.php:714]: Good coverage

**Review by pasquinis — CHANGES_REQUESTED** [review]
For me is ok, but look my comments and evaluate it

**Comment by lucacervello** [comment]
@pasquinis new commit, new review. :smiley:

**Review by pasquinis — APPROVED** [review]
Now is good :+1:

**Pull request merged**